frontend/www/js/omegaup/arena/arena.js

```diff
@@ -924,29 +952,25 @@ export class Arena {
           self.problems[alias].languages !== ''
         ) {
           const currentPoints = parseFloat(self.problems[alias].points || '0');
-          let index = self.elements.navBar.problemsList.findIndex(
+          let index = self.elements.navBar.problems.findIndex(
```

Comment:
```suggestion
          const currentProblem = self.elements.navBar.problems.find(problem => problem.alias === alias);
          currentProblem.bestScore = problem.points;
          currentProblem.maxScore = currentPoints;
```

ya no necesitamos el índice!

Reply:
Listo